### ISSUE TITLE
Argparse bug in capture_activations.py

### ISSUE THREAD

**Issue by lintweaker** [dscripka/openWakeWord#186]
It seems there is a bug in the example 'capture_activations.py' script. There are two
```
parser=argparse.ArgumentParser()
```
statements making it only use the arguments from the last statement. This is at least an issue with tested Python3.9.2 on my Pi

**Comment by dscripka** [comment]
You are correct, the arg parsing is broken for that script. Will fix it for the next release.
